core/validation/specific_validations/resources/TF-grant-awarded.csv

```diff
@@ -1,812 +1,812 @@
-TD-STE-06,2000000
-TD-STE-07,10000000
-TD-STE-08,3500000
-TD-STE-09,3750000
-TD-STF-01,4234943
-TD-STF-02,1211193
-TD-STF-03,2151765
-TD-STF-04,3080864
-TD-STF-05,4165870
-TD-STF-06,1234681
-TD-STF-08,3585682
-TD-STF-09,1739702
-TD-STF-10,195300
-TD-STH-02,7240000
-TD-STH-03,890000
-TD-STH-05,3250000
-TD-STH-06,2500000
-TD-STH-07,1250000
-TD-STH-08,1370000
-TD-STH-09,5500000
-TD-STH-10,1703000
-TD-STH-11,1297000
-TD-STI-01,3004249
-TD-STI-02,345538
-TD-STI-04,2955000
-TD-STI-05,751260
-TD-STI-06,866800
-TD-STI-08,3198788
-TD-STI-09,1477500
-TD-STI-10,5061915
-TD-STI-11,1940450
-TD-STI-12,298500
-TD-STO-01,350000
-TD-STO-02,475044
-TD-STO-03,602990
-TD-STO-04,739350
-TD-STO-05,2700000
-TD-STO-06,4745790
-TD-STO-07,9543166
-TD-STO-08,1147330
-TD-STO-09,2300000
-TD-STO-10,350000
-TD-STO-11,1146330
-TD-STP-01,4991000
-TD-STP-02,1623000
-TD-STP-03,4813000
-TD-STP-04,1000000
-TD-STP-05,777000
-TD-STP-06,7021000
-TD-STP-07,875000
-TD-STV-01,4146545
-TD-STV-02,2148000
-TD-STV-03,513700
-TD-STV-04,615000
-TD-STV-05,934000
-TD-STV-06,117000
-TD-STV-07,5974980
-TD-STV-08,2930400
-TD-STV-10,5834400
-TD-STV-11,1437700
-TD-STV-12,418275
-TD-SWI-01,6000000
-TD-SWI-02,3000000
-TD-SWI-03,5000000
-TD-SWI-04,5000000
-TD-SWI-05,400000
-TD-SWI-06,100000
-TD-TEL-01,17025000
-TD-TEL-03,2637500
-TD-TEL-04,2637500
-TD-THO-01,4560000
-TD-THO-02,10640000
-TD-THO-03,4465000
-TD-THO-04,3040000
-TD-THO-05,1195000
-TD-TIL-01,379000
-TD-TIL-02,5071000
-TD-TIL-03,6600000
-TD-TIL-04,10750000
-TD-TOD-01,3525000
-TD-TOD-02,1207929
-TD-TOD-03,1138000
-TD-TOD-04,819928
-TD-TOD-05,3873000
-TD-TOD-06,3244194
-TD-TOD-07,1735552
-TD-TOD-08,1081397
-TD-TOD-11,875000
-TD-TOR-01,11042500
-TD-TOR-03,2000000
-TD-TOR-04,2500000
-TD-TOR-05,600000
-TD-TOR-06,2000000
-TD-TOR-07,3000000
-TD-TOR-09,600000
-TD-TOR-10,157500
-TD-TRU-01,10409252
-TD-TRU-02,5215106
-TD-TRU-03,952705
-TD-TRU-04,1905412
-TD-TRU-05,952705
-TD-TRU-06,3334820
-TD-TRU-07,476000
-TD-TRU-08,354000
-TD-WAK-01,9980000
-TD-WAK-02,7400684
-TD-WAK-04,6514268
-TD-WAK-05,240000
-TD-WAL-01,1314000
-TD-WAL-02,909000
-TD-WAL-03,968060
-TD-WAL-04,908000
-TD-WAL-05,405669
-TD-WAL-06,1783000
-TD-WAL-07,4493000
-TD-WAL-08,904040
-TD-WAL-09,639000
-TD-WAL-10,8976231
-TD-WAR-01,3090250
-TD-WAR-02,5525000
-TD-WAR-03,4947650
-TD-WAR-04,976600
-TD-WAR-05,2872200
-TD-WAR-06,812300
-TD-WAR-07,3621000
-TD-WAR-08,255000
-TD-WES-01,2700000
-TD-WES-02,42600
-TD-WES-03,13000000
-TD-WES-04,1220000
-TD-WES-05,6930000
-TD-WES-06,1100000
-TD-WHI-01,2600000
-TD-WHI-02,823051
-TD-WHI-03,2300000
-TD-WHI-04,1000000
-TD-WHI-05,10000000
-TD-WHI-06,237000
-TD-WHI-07,139948
-TD-WOL-01,5000000
-TD-WOL-02,1000000
-TD-WOL-03,498000
-TD-WOL-04,5247657
-TD-WOL-05,3282343
-TD-WOL-07,500000
-TD-WOL-08,2972000
-TD-WOL-09,5000000
-TD-WOL-10,1500000
-TD-WRC-01,1960000
-TD-WRC-02,2500000
-TD-WRC-03,10000000
-TD-WRC-04,640000
-TD-WRC-05,4500000
-TD-WRK-01,4500000
-TD-WRK-02,8200000
-TD-WRK-03,4000000
-TD-WRK-04,5400000
-TD-WRK-05,1000000
+﻿Index Code,RDEL,CDEL,Total
```

Comment:
Is this file coming straight from TF, or are we extracting the data from elsewhere and putting it into a csv? If the latter then we should probably get this checked over by e.g. Ade

Reply:
This data is coming straight from TF with the aim to have them take full ownership of it in the future

Reply:
I've justed added a test called test_validate_funding_spent_no_errors that does this